pkg/component/runtime/command.go

```diff
@@ -243,11 +257,26 @@ func (c *CommandRuntime) start(comm Communicator) error {
 		return nil
 	}
 	cmdSpec := c.current.Spec.Spec.Command
-	env := make([]string, 0, len(cmdSpec.Env))
+	env := make([]string, 0, len(cmdSpec.Env)+2)
```

Comment:
Why the magic +2? Can you add a comment or make it a constant?

Reply:
Probably the component ID and type being added below?

Reply:
Yes its from the extra 2 added below. I think making it a constant would pull the reason for the additional space away from the code.